content/applications/websites/website/reporting/analytics.rst

```diff
@@ -137,7 +137,9 @@ measurement codes and related code fragments, collectively known as tags on your
 app, directly through the code injector.
 
 .. warning::
-   Google Tag Manager may not be compliant with local data protection regulations.
+   - GTM integration is possible in Odoo standard, although some data layers might not be installed
+     (e.g. advanced eCommerce tracking data layers).
```

Comment:
What if we said this instead (copied from your explanation); I think by providing a little bit more context and explanations, it makes it easier for people to understand?

```suggestion
   - Some GTM tags use data layers (e.g., advanced eCommerce tracking data layers) to retrieve variables and send them to Google Analytics. Data layers are currently not managed in Odoo. 
```
(I changed it a little bit, just trying to make it as complete as possible while staying as concise as possible; is it still accurate?)

Reply:
Hi, it seems accurate to me :) !